src/Entity/Event.php

```diff
@@ -543,4 +530,47 @@ public function setMixDeskScenography(?int $mixDeskScenography): self
 
         return $this;
     }
+
+    public function getToken(): ?string
+    {
+        return $this->token;
+    }
+
+    public function setToken(string $token): self
+    {
+        $this->token = $token;
+
+        return $this;
+    }
+
+    /**
+     * @Assert\IsTrue(message="Пожалуйста, проверьте указанные даты: событие не может закончиться до своего начала.")
+     */
+    public function isEndDateSane()
+    {
+        return $this->getStartDate() <= $this->getEndDate();
```

Comment:
@daria-sieroshtan 
I think it's OK to have single-day event, with start and end dates being equal
PS. Meh

Reply:
That's why there is "less than or equal", not "less"

Reply:
That's why there is "less than or equal", not "less than"